### DISCUSSION TITLE
Using Amiberry on a BananPi Zero M2 running Armbian as bootable first app?

### DISCUSSION THREAD

**Discussion by theshinyknight** [BlitterStudio/amiberry#1128] [General]
I would like to make a quick bootable Amiga system using a BananaPi Zero M2 board; which is a clone of a Raspberry Pi Zero 2 basically.

The problem is that it does not run Raspbian, but a different flavor OS called Armbian. I can get the source and try compiling on it but I was wondering if anyone tried already on one of these boards, and was able to get to the point where they can just boot directly into Amiberry, as I would like to make the system boot as if it was an Amiga, and control everything via console, so no WM or GUI installed to keep the system light (the board has only 512 MB of ram, so text only interface is preferred).

Can this be done? Or to run Amiberry I need to have the WM and GUI running anyway? I don't plan to press F12 to change options that often, and ideally would change them via text interface. Thanks

**Comment by midwan** [comment]
Yes, it can be done. You don't need a WM to get the GUI or emulation screen showing, but you do need to have support for your back-end in SDL2 (for example, KMSDRM or Wayland or whatever will be used).
Then it's a matter of putting it in the startup apps of your system, for example you could have it running as a service.
But all this is a bit out of scope for me, perhaps someone from the Armbian community can assist with the details.

**Comment by theshinyknight** [comment]
Thanks; was able to build and boot it on a build of pure text based Armbian.

The interface run, although I am not sure how can I optimize the output. Is there a config file for the Amiberry interface? I would like to set it up so it can output at something like 800x600 to save resources; but the only thing I see is just the amiberry UI executable, so not sure if there is a way to skip the whole loading of the actual UI and tell Amiberry to load directly a hd file with WB 3.1 for example, and load directly a configuration.

Have been reading the wiki docs but I don't see specific commands you can pass via "cli"; only the ones you give via UI. Thanks!

**Comment by solskogen** [comment]
Have you tried` ./amiberry --help` ? :-)

**Comment by theshinyknight** [comment]
I should have tried that first :D Thanks! It does work as planned now 🥇
